app/models/UserSchema/UserSchema.js

```diff
@@ -0,0 +1,47 @@
+import mongoose from "mongoose";
+
+const UserSchema = new Schema({
+  email: {
+    type: String,
+    required: [true, "Email is required"],
+    lowercase: true,
+    trim: true,
+    unique: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    minLength: [3, "At least 3 characters"],
+  },
+  city: {
+    type: String,
+    required: true,
+    minLength: [3, "At least 3 characters"],
+  },
+  phone: {
+    type: Number,
+    required: true,
```

Comment:
For me, as we use `e-mail` as a login element (so, it is required) we shouldn't require `phone number`